ConsoleApp10/ConsoleApp10/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+class Program
+{
+    static Random rnd = new Random();
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Введите количество строк массива в пределах от 5 до 20");
+        Console.WriteLine("------------------------------------------------------");
+        var rowNumber = int.Parse(Console.ReadLine());
+        Console.WriteLine("Введите количество столбцов массива в пределах от 5 до 20");
+        Console.WriteLine("---------------------------------------------------------");
+        var columnNumber = int.Parse(Console.ReadLine());
+
+        var MMM = new int[rowNumber, columnNumber];
```

Comment:
Локальные переменные принято называть с маленькой буквы
```suggestion
        var mmm = new int[rowNumber, columnNumber];
```